### PR TITLE
Add configurable brushes

### PR DESCRIPTION
**Pull request by danbarragan**
Adds configurable brushes and includes white as a color so I think this technically fixes:

- https://github.com/danbarragan/drawydraw/issues/26
- https://github.com/danbarragan/drawydraw/issues/27 (just use the big brush?)

Since I was partying on drawing code I also fixed https://github.com/danbarragan/drawydraw/issues/25

image

### CODE REVIEW

**Comment by danbarragan** [comment]
@mick-warehime

File: client/src/components/DrawingScreen/BrushConfig/BrushConfig.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  Greenery: '#88B04B',
+  Coral: '#FF6F61',
+  Violet: '#6B5B95',
+  Marsala: '#955251',
+  Orchid: '#B565A7',
+  Turquoise: '#5B8AC',
+  Mimosa: '#EFC050',
+});
+
+const Widths = Object.freeze({
```
Comment (mick-warehime) [inline client/src/components/DrawingScreen/BrushConfig/BrushConfig.jsx:20]: nit: BrushSize here and throughout?

width.name is a bit hard to parse farther down

Comment (danbarragan) [reply client/src/components/DrawingScreen/BrushConfig/BrushConfig.jsx:20]: makes sense!

**Comment by danbarragan** [comment]
> wow. this is awesome.
> 
> i think moving the white brush to just be an eraser will be more obvious but that can be fixed later

sounds good! We _might_ also want to reconsider a big clear all button since if you keep erasing and drawing the strokes array will become yuuuuge ahahaha
